### PR TITLE
python3Packages.pypykatz: init at 0.3.15

### PR DESCRIPTION
**Pull request by fabaff**
###### Motivation for this change
Mimikatz implementation in Python

https://github.com/skelsec/pypykatz

Also, contains [`msldap`](https://github.com/skelsec/msldap) which is a Python LDAP library for auditing MS AD and a dependency for `pypykatz` and [`lsassy`](https://github.com/Hackndo/lsassy), a Python module to extract data from Local Security Authority Subsystem Service (LSASS).

Related to #81418

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/msldap/default.nix [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+buildPythonPackage rec {
+  pname = "msldap";
+  version = "0.3.22";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1ivq56953skql8f255nqx2sg4mm0kz2pr5b4dx62dx7jdgd1xym3";
+  };
+  propagatedBuildInputs = [
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/msldap/default.nix:21]:
```suggestion
  };

  propagatedBuildInputs = [
```

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 109005` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>9 packages built:</summary>
  <ul>
    <li>python37Packages.lsassy</li>
    <li>python37Packages.msldap</li>
    <li>python37Packages.pypykatz</li>
    <li>python38Packages.lsassy</li>
    <li>python38Packages.msldap</li>
    <li>python38Packages.pypykatz</li>
    <li>python39Packages.lsassy</li>
    <li>python39Packages.msldap</li>
    <li>python39Packages.pypykatz</li>
  </ul>
</details>

File: pkgs/development/python-modules/msldap/default.nix (hunk 1 above)
Comment (fabaff) [reply pkgs/development/python-modules/msldap/default.nix]: Thanks, fixed.

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review which does not build all packages (e.g. lumo, tensorflow or pytorch)
If you find some bugs or got suggestions for further things to search or run please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 109005` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
9 packages built:
</summary>
<ul>
<li>python37Packages.lsassy</li>
<li>python37Packages.msldap</li>
<li>python37Packages.pypykatz</li>
<li>python38Packages.lsassy</li>
<li>python38Packages.msldap</li>
<li>python38Packages.pypykatz</li>
<li>python39Packages.lsassy</li>
<li>python39Packages.msldap</li>
<li>python39Packages.pypykatz</li>
</ul>
</details>

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review which does not build all packages (e.g. lumo, tensorflow or pytorch)
If you find some bugs or got suggestions for further things to search or run please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 109005` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
9 packages built:
</summary>
<ul>
<li>python37Packages.lsassy</li>
<li>python37Packages.msldap</li>
<li>python37Packages.pypykatz</li>
<li>python38Packages.lsassy</li>
<li>python38Packages.msldap</li>
<li>python38Packages.pypykatz</li>
<li>python39Packages.lsassy</li>
<li>python39Packages.msldap</li>
<li>python39Packages.pypykatz</li>
</ul>
</details>

**Pull request merged**